doc/releases/mimic.rst

```diff
@@ -69,8 +69,39 @@ Notes
 Instructions
 ~~~~~~~~~~~~
 
+#. If your cluster was originally installed with a version prior to
+   Luminous, ensure that it has completed at least one full scrub of
+   all PGs while running Luminous.  Failure to do so will cause your
+   monitor daemons to refuse to join the quorum on start, leaving them
+   non-functional.
+
+   If you are unsure whether or not your Luminous cluster has
+   completed a full scrub of all PGs, you can check your cluster's
+   state with the following steps:
+
+   * Retrieve your OSD map: ``ceph osd getmap -o  /tmp/osdmap``
+   * Examine it: ``osdmaptool --print /tmp/osdmap``
```

Comment:
These two steps can be combined into "ceph osd dump | grep ^flags", right?

Reply:
Absolutely. I had been looking for a way to abbreviate this but didn't find one. :)

Fixed that. Also, rebased this PR on current master.